grid2op/Runner/Runner.py

```diff
@@ -663,9 +688,11 @@ def _run_one_episode(env, agent, logger, indx, path_save=None,
                 cum_reward += reward
                 time_step += 1
                 pbar_.update(1)
-
+                opp_attack = env.oppSpace.last_attack
```

Comment:
add a description somewhere (in the opponent space for example) that the last_attack attribute should be necessarily updated (if not done already)